sharding-core/src/main/java/io/shardingsphere/core/keygen/DefaultKeyGenerator.java

```diff
@@ -86,23 +88,24 @@
     
     private long lastMilliseconds;
     
-    /**
-     * Set work process id.
-     * 
-     * @param workerId work process id
-     */
-    public static void setWorkerId(final long workerId) {
-        Preconditions.checkArgument(workerId >= 0L && workerId < WORKER_ID_MAX_VALUE);
-        DefaultKeyGenerator.workerId = workerId;
+    @Override
+    public void setKeyGeneratorProperties(final Properties properties) {
```

Comment:
Consider about rename method to `setProperties`, because the class name is `KeyGenerator`, related about context, `setProperties` is clear and meaningful.